app/domain/errors.rb

```diff
@@ -171,10 +171,20 @@ module AuthnOidc
       module AuthnIam
 
         InvalidAWSHeaders = ::Util::TrackableErrorClass.new(
-          msg:  "'Invalid or expired AWS headers: {0}",
+          msg:  "Invalid or expired AWS headers: {0}",
           code: "CONJ00018E"
         )
 
+        VerificationError = ::Util::TrackableLogMessageClass.new(
+          msg:  "Verification of IAM identity failed with exception: {0-exception}",
+          code: "CONJ00063E"
+        )
+
+        VerificationErrorCode = ::Util::TrackableLogMessageClass.new(
```

Comment:
I suggest you'll use more meaningful names, as our approach of separating the log text to a separate files means that the developer is not aware of what is exactly the text from first glance